models/multiphase/d3q27_pf_velocity/Dynamics.c.Rt

```diff
@@ -320,11 +314,17 @@ CudaDeviceFunction vector_t calcGradPhi(){
 }
 
 CudaDeviceFunction void calcPhaseF(){
+	// This function is run as the second stage of each action and fills
```

Comment:
I think docstrings for C/C++ functions are conventionally placed above the function they are describing with /* */ as comments. So this should be (also applies to other function where docsting was added) 

```cpp
/* This function is run as the second stage of each action and fills the iteration
buffer with updated values of PhaseF */
CudaDeviceFunction void calcPhaseF(){
     ....
```